arch/xtensa/src/esp32/Kconfig

```diff
@@ -859,6 +859,7 @@ config ESP32_RTC_HEAP
 config ESP32_IRAM_HEAP
 	bool "Use the rest of IRAM as a separate heap"
 	select ARCH_HAVE_EXTRA_HEAPS
+	select LIBC_ARCH_ATOMIC
```

Comment:
LIBC_ARCH_ATOMIC has been removed while ago. am i missing something?

Reply:
Was add in this PR https://github.com/apache/nuttx/pull/14803

Reply:
Shouldn't it be enabled for ESP32-S2 and ESP32-S3 too?

Reply:
ESP32-S2 and ESP32-S3 do not support IRAM HEAP, so there is no need